__tests__/app.test.js

```diff
@@ -111,4 +112,192 @@ describe('GET requests', () => {
             })
         });
     });
+
+    describe('GET /api/articles/:article_id/comments', () => {
+        test('200 - responds with array of comments for the given article_id', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.comments).toHaveLength(11)
+                body.comments.forEach((comment) => {
+                    expect(comment).toHaveProperty("comment_id", expect.any(Number))
+                    expect(comment).toHaveProperty("votes", expect.any(Number))
+                    expect(comment).toHaveProperty("created_at", expect.any(String))
+                    expect(comment).toHaveProperty("author", expect.any(String))
+                    expect(comment).toHaveProperty("body", expect.any(String))
+                    expect(comment).toHaveProperty("article_id", 1)
+                }) 
+                expect(body.comments).toBeSorted({ key: "created_at", descending: true })
+            })
+        })
+        test('200 - responds with message when given article_id valid but has no comments', () => {
+            return request(app)
+            .get('/api/articles/2/comments')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.msg).toBe('article_id has no comments')
```

Comment:
This will currently, literally print out article_id rather than the article_id number